runtime/doc/usr_05.txt

```diff
@@ -387,6 +387,27 @@ Only files of this filetype will use the settings from the plugin.  The <name>
 part of the plugin file doesn't matter, you can use it to have several plugins
 for the same filetype.  Note that it must end in ".vim".
 
+THE PLUG COMMAND                                                    *:plug*
+
+:plug[!] {spec}	        Add the plugin bundle defined by {spec} to 'runtimepath'.
+			{spec} can be a full path, the name of a directory in
+			the plugins folder (defined as `~/.vim/plugged/` by
+			default, see 'plugindir'), or it can have the form
+			`PREFIX/DIRNAME`.
+                        If {spec} is a full path, the command will work only
+                        if it exists in the filesystem.
+			If [!] is given, source the *.vim files in the plugin/
+			folder within {spec}.
+
+You can declare which paths to load plugins from using |:plug| from your
+|vimrc| file: >
+
+	plug ~/.config/nvim/plugged/nvim-plugin
+	plug neovim/nvim-plugin
+	plug nvim-plugin
+
+Note These are all equivalent on a default setup.
```

Comment:
Missing colon after `Note`: without it sentense like `Note These` looks rather strange. Also better to use `All these commands are equivalent`.